src/main/java/nextstep/path/config/FinderConfig.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.path.config;
+
+import nextstep.path.service.DijkstraShortestPathService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class FinderConfig {
+
+    private final String DIJKSTRA = "dijkstra";
```

Comment:
상수를 의도하신 거라면 관례에 따라 `static` 키워드까지 추가해보면 어떨까요?
```suggestion
    private static final String DIJKSTRA = "dijkstra";
```

Reply:
제가 미처 생각을 못했네요. 좋은 의견 감사합니다!!!